source/connect/network-compression.txt

```diff
@@ -0,0 +1,103 @@
+.. _pymongo-network-compression:
+
+========================
+Compress Network Traffic
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: zlib, zstandard, snappy
+
+{+driver-short+} provides a connection option to compress messages, which reduces the amount
+of data passed over the network between MongoDB and your application.
+
+{+driver-short+} supports the following compression algorithms.
+
+1. `Snappy <https://google.github.io/snappy/>`__: Available in MongoDB 3.6 and later. This
+   option requires the `python-snappy <https://pypi.org/project/python-snappy/>`__ package.
+
+2. `Zlib <https://zlib.net/>`__: Available in MongoDB 3.6 and later. This option requires
+   the zlib module, included in the standard library in Python v1.5 and later.
+
+3. `Zstandard <https://github.com/facebook/zstd/>`__: Available in MongoDB 4.2 and later.
+   This option requires the `zstandard <https://pypi.org/project/zstandard/>`__ package.
+
+If you don't specify a compression algorithm, {+driver-short+} doesn't compress your
+network traffic. If you specify multiple compression algorithms, the driver selects the
+first one in the list supported by your MongoDB instance.
+
+.. _pymongo-enable-compression:
+
+Specify Compression Algorithms
+------------------------------
+
+To enable compression for the connection to your MongoDB instance,
+specify the algorithms you want to use in one of the following ways: 
+
+You can do this in two ways: by passing an argument to the
+``MongoClient`` constructor or through a parameter in your connection string.
+
+.. tabs::
+
+   .. tab:: MongoClient
+      :tabid: mongoclient
+
+      .. code-block:: python
+         :emphasize-lines: 2
+
+         client = pymongo.MongoClient("mongodb://<username>:<password>@<hostname@:<port>",
+                                      compressors="snappy,zstandard,zlib")
+
+   .. tab:: Connection String
+      :tabid: connectionstring
+
+      .. code-block:: python
+         :emphasize-lines: 2
+
+         uri = ("mongodb://<username>:<password>@<hostname>:<port>/?"
+                "compressors=snappy,zstandard,zlib")
+         client = pymongo.MongoClient(uri)
+
+If you specify ``zlib`` as one of your compression algorithms, you can also use the
+``zlibCompressionLevel`` option to specify a compression level. This option accepts
+an integer value between ``-1`` and ``9``:
+
+- **-1:** (Default). zlib uses its default compression level (usually ``6``).
+- **0:** No compression.
+- **1:** Fastest speed but lowest compression.
+- **9:** Best compression but slowest speed.
```

Comment:
I'm not attached to this list style

Reply:
I think it's fine; you could also use a table with two columns

Reply:
i went to a working group yesterday and one of the things we talked about was the overuse of tables. probably why i went with a list here